PackageIndexer/DotnetPackageIndex.cs

```diff
@@ -41,12 +41,12 @@ public static async Task CreateAsync(string packageListPath, bool usePreviewVers
         params string[] feedUrls
         )
     {
-        var packages = new List<PackageIdentity>();
+        var packages = new List<(PackageIdentity, bool)>();
```

Comment:
Throughout this logic, I'd consider named tuples. I think it will make it easier to read:

```suggestion
        var packages = new List<(PackageIdentity version, bool isDeprecated)>();
```

Reply:
Thanks @BillWagner, I like that much better. Just never used them before!